.travis.yml

```diff
@@ -6,9 +6,17 @@ sudo: false
 cache:
   directories:
   - node_modules
+
 before_install:
 # Symlink closure library
 - ln -s $(npm root)/google-closure-library ../closure-library
+
+before_script:
+  - export DISPLAY=:99.0
+  - if [ "${TRAVIS_OS_NAME}" == "linux" ]; then ( tests/scripts/setup_linux_env.sh  ) fi
+  - if [ "${TRAVIS_OS_NAME}" == "osx" ]; then ( tests/scripts/setup_osx_env.sh ) fi
```

Comment:
We can exclude `osx` builds since we don't build for them in our matrix.

Reply:
Fixed 2cbed37